src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -479,99 +474,6 @@ private void deleteCourses(Collection<CourseAttributes> courses) {
         }
     }
 
-    //TODO: remove this when we confirm it is not needed
```

Comment:
Eclipse did this. Apparently they don't care about the `@SuppressWarnings("unused")`.